common/src/main/java/tachyon/util/io/BufferUtils.java

```diff
@@ -73,7 +73,9 @@ public static void cleanDirectBuffer(ByteBuffer buffer) {
       }
       final Object cleaner = sByteBufferCleanerMethod.invoke(buffer);
       if (cleaner == null) {
-        LOG.error("Failed to get cleaner for ByteBuffer");
+        if (buffer.capacity() > 0) {
```

Comment:
i guess that is different from what @siyangy is having here.
In the current code, as long as cleaner is null, we return.